### ISSUE TITLE
Can you show me how to train your code?

### ISSUE THREAD

**Issue by markofan** [ehosseiniasl/3d-convolutional-network#5]
I want to use it to classify AD and Normal, but I have some trouble of how to train your code, can you tell me how to do that?

**Comment by markofan** [comment]
what is the difference between finetune_scae_crossvalidate and get_hidden_finetuned?

**Comment by ehosseiniasl** [comment]
1- finetune_scae_crossvalidate() is for finetuning the scae model using cross validation!
2- get_hidden_finetuned() is run after training the scae model to extract the hidden features of all layers for test set!
